SConstruct

```diff
@@ -1496,10 +1499,8 @@ if testEnv["PLATFORM"] == "darwin" :
 def makeSymLinks( env, target ) :
 
 	links = {
-		"$IECORE_MAJORMINORPATCH_VERSION" : "$IECORE_MAJORMINOR_VERSION",
```

Comment:
I tried to understand why there were duplicates of these before, one with and one without braces. That led me to 3cd9db2458e44f75e1b587e3c9ef07e0f123b6b2 but didn't explain it. I presume it's OK to lose the duplicates, but do you know why we had them in the first place?

Reply:
No idea why we had them. My best guess is that was 2011 and I didn't understand SCons very well at the time (if I do now...)